sass/_sponsors.scss

```diff
@@ -0,0 +1,31 @@
+#sponsorSection {
+  text-align: center;
+  display: block;
+  padding: 20px;
+}
+
+#sponsorSection h2 {
+  font-family: SaxMono;
+  font-size: 62px;
+}
+
+.sponsorLogo img {
+ padding: 10px;
+}
+
+#sponsorSection p{
+  font-family: lato;
```

Comment:
Here's a font stack for this.
```
    font-family: Lato, Calibri, Optima, Arial, sans-serif;
```
At the very least you should always end font-family with a default system fallback (`sans-serif`, `serif`, `monospace`, `fantasy`, `cursive`). The browser will consult the OS for what the default system font is for that type of font in case the user doesn't have the one you specified installed, or if it's a web font (like Lato) if it doesn't load immediately or at all, it will fallback to a similar type of font.